client/apps/plugin-status/health.js

```diff
@@ -8,53 +8,27 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import Indicator from './indicator';
+import WooCommerceServicesIndicator from './woocommerce-services-indicator';
 import SettingsGroupCard from 'woocommerce/woocommerce-services/components/settings-group-card';
 
-const HealthView = ( { translate, moment, healthItems } ) => {
```

Comment:
Tried to simplify this view by encapsulating most of the logic into a new `WooCommerceServicesIndicator` React component